base/c.jl

```diff
@@ -18,7 +24,8 @@ Equivalent to the native `char` c-type.
 """
 Cchar
 
-if is_windows()
+# The ccall is equivalent to Sys.iswindows(), but that's not defined yet
```

Comment:
Comment no longer applies. I'll remove that in a separate, CI skipped commit once the current CI run finishes, since removing the comment is a non-functional change and is not worth rerunning CI over.